staging/src/k8s.io/apiserver/pkg/util/flowcontrol/apf_controller.go

```diff
@@ -52,6 +55,8 @@ import (
 	fclistersv1a1 "k8s.io/client-go/listers/flowcontrol/v1alpha1"
 )
 
+const timeFmt = "2006-01-02T15:04:05.999"
```

Comment:
i suppose it's only for logging, why not calling `Time.MarshalJSON` instead?

Reply:
Yes, this is only used in log messages emitted at V level 5 --- which is used in debugging but not production.

Note also that the values at hand are of type `time.Time` rather than kube's own wrapper type.

I could have used `time.RFC3339Nano` directly.  In fact, I did at first.  But the random precision wore out its welcome.  I did not need the endless repetitions of the time zone either.